requirements/constraints.txt

```diff
@@ -70,11 +70,8 @@ python3-saml==1.5.0
 # CR-1912: Check if super-csv 0.9.7 is breaking grade export
 super-csv==0.9.6
 
-# Bumping requests-oauthlib to 1.2 updates oauthlib to 3.0.0, which changes a response code in certain cases
-# This causes a test to fail. Before fixing, we would need to make sure mobile apps are functional with this change
-# See https://github.com/oauthlib/oauthlib/issues/264 for more context
-requests-oauthlib==1.1.0
-oauthlib==2.1.0
+# oauthlib>3.0.1 causes test failures
+oauthlib==3.0.1
```

Comment:
It looks like you lost the actual upgrades in the output requirements files during rebasing.

Reply:
`requests-oauthlib` has latest version `1.3.0` so I removed the constraint to move to the latest possible version and oauthlib has been upgraded to `3.0.1`.

Reply:
oh okay, I think I'll need to run make upgrade again.